clang/lib/AST/Interp/ByteCodeExprGen.h

```diff
@@ -54,7 +54,7 @@ class ByteCodeExprGen : public ConstStmtVisitor<ByteCodeExprGen<Emitter>, bool>,
 public:
   /// Initializes the compiler and the backend emitter.
   template <typename... Tys>
-  ByteCodeExprGen(Context &Ctx, Program &P, Tys &&... Args)
+  ByteCodeExprGen(Context &Ctx, Program &P, Tys &&...Args)
```

Comment:
This and the next one are irrelevant whitespace changes.

Reply:
This is how our clang-format config wants this to look like. If I add a white-space there it will no longer conform to our formatting standards.

Reply:
Yeah but clang-format shouldn't touch those lines since you didn't touch them either.

Reply:
It was probably the autoformat from my editor then. I removed both format changes.